core/profiles/standard/tests/standard.test

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @file
+ * Tests for the Standard Installation Profile.
+ */
+
+
+/**
+ * Verifies that tests bundled with installation profile modules are found.
+ */
+class StandardInstallTestCase extends BackdropWebTestCase {
+  protected $profile = 'standard';
+
+  /**
+   * Tests existence of test case located in an installation profile module.
+   */
+  function testRolesExist() {
+    $user_1 = user_load(1);
+    $this->assertTrue(user_has_role('administrator', $user_1), 'User ID 1 is an administrator');
+
+    // Create a user assigned to the editor role.
+    $edit = array();
+    $edit['name']   = $this->randomName();
```

Comment:
I know it looks pretty, but our coding standards (and my OCDs) say no 😄 

```suggestion
    $edit['name'] = $this->randomName();
```

Reply:
Yeah, unintentional. Copy/paste from elsewhere. :P